lib/renderer.js

```diff
@@ -1261,10 +1261,11 @@
         // 10. The layer is not hidden.
```

Comment:
please update the detailed comments above - especially with the weirdness of scaled layerComps